### ISSUE TITLE
[Bug]: It’s too late to use the Mica DWM function

### ISSUE THREAD

**Issue by emako** [Simnico99/MicaWPF#105]
### Describe the bug

Short white screen in dark mode

### Steps to reproduce the bug

Startup

### Expected behavior

Apply mica on `OnSourceInitialized`:

```c#
    protected override void OnSourceInitialized(EventArgs e)
    {
        base.OnSourceInitialized(e);

        // Apply mica here!
    }
```

### Windows version

Windows 11 version 23H2 (22631, 2023 Update)

### .Net Version

.NET 8.x

### .MicaWPF Version

MicaWPF, MicaWPF.Lite

**Comment by Simnico99** [comment]
Should be fixed in version 6.2.9 tell me if it works I've move the logic to OnSourceInitialized

**Comment by moob158** [comment]
I have a problem with this change.

I have WindowStyle set to "none" and mica style is not apply to the window

work great in 6.2.8 version
